tests/conftest.py

```diff
@@ -903,50 +904,18 @@ def is_port_in_use(host: str, port: int) -> bool:
         return s.connect_ex((host, port)) == 0
 
 
-class BaseP2PLibp2pTest:
-    """Base class for p2p libp2p tests"""
-
-    cwd: str
-    t: str
-    tmp_dir: str
-    log_files: List[str] = []
-    multiplexers: List[Multiplexer] = []
-    capture_log = True
-
-    @classmethod
-    def setup_class(cls):
-        """Set the test up"""
-        cls.cwd, cls.t = os.getcwd(), tempfile.mkdtemp()
-        cls.tmp_dir = os.path.join(cls.t, "temp_dir")
-        Path(cls.tmp_dir).mkdir(exist_ok=True)
-        os.chdir(cls.t)
-        atexit.register(cls.teardown_class)
-
-    @classmethod
-    def teardown_class(cls):
-        """Tear down the test"""
-        logger.debug(f"Cleaning up {cls.__name__}")
-        for mux in cls.multiplexers:
-            mux.disconnect()
-        cls.multiplexers.clear()
-        cls.log_files.clear()
-        os.chdir(cls.cwd)
-        remove_test_directory(cls.t)
-        logger.debug(f"Teardown of {cls.__name__} successful")
-
-
 def _make_libp2p_connection(
-    data_dir: str,
+    data_dir: str = DEFAULT_LIBP2P_TEST_DIR,
```

Comment:
this ensures we use the same base for temporary test directories and always remove them - thus far directories in `/tmp/` were accumulating each time tests were ran